### PR TITLE
[team_test_0]: First PR HelloWorld

### PR DESCRIPTION
**Pull request by King2Light**
### **What?**

- brief: test helloworld。
- author: King2Light

### What changes are included in this

- 新建团队和个人目录
- HelloWorld源码文件

### Are these changes tested?
测试HelloWorld程序，结果如下图所示
微信截图_20230617170829

### CODE REVIEW

**Comment by Light-City** [comment]
> What? brief: 这是一个测试pr，用于帮助大家快速按照这个模版提交。 author: Light What changes are included in this PR? 包含了一个团队目录、个人目录、个人学习内容。
> 
> Are these changes tested? 测试代码见HelloWorld.cpp，里面编写了一个打印Hello World程序 微信截图_20230617170829

format it into markdown. use template.

**Comment by Light-City** [comment]
1.The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

2.your title style is wrong

**Comment by King2Light** [comment]
> 1.The directory structure is incorrect, you can refer to what others have done, or this document
> 
> https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md
> 
> 2.your title style is wrong

I have modified,please check again.

File: stl/team_light/king2light/src/HelloWorld.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+using namespace std;
```
Comment (martinx) [inline stl/team_light/king2light/src/HelloWorld.cpp:3]: Do not use a using-directive to make all names from a namespace available.

File: stl/team_light/king2light/src/HelloWorld.cpp [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+int main(void)
+{
```
Comment (martinx) [inline stl/team_light/king2light/src/HelloWorld.cpp:6]: `{` do not start a new line

**Review by martinx — COMMENTED** [review]
Google Code Style: https:google.github.io/styleguide/cppguide.html#Namespaces

**Pull request closed without merging**